Sources/HttpPipeline/SharedMiddlewareTransformers.swift

```diff
@@ -185,7 +185,7 @@ public func requestLogger(logger: @escaping (String) -> Void)
         return middleware(conn).flatMap { b in
           IO {
             let endTime = Date().timeIntervalSince1970
-            logger("[Request] \(conn.request.httpMethod ?? "GET") \(conn.request)")
+            logger("[Request] \(conn.request.httpMethod ?? "GET") \(conn.request.url?.absoluteString ?? "")")
```

Comment:
this is necessary cause linux prints out urls in a weird way, c.f. https://github.com/apple/swift-corelibs-foundation/pull/1312